src/asp.jl

```diff
@@ -140,3 +140,10 @@ function post_asp_tsvd(path, At, yt, Av, yv)
 
    return _post.(path)
 end
+
+function select(tracer, solver, A, y) #can be called by the user to warm-start the selection
```

Comment:
I feel this should be of the form 
```
select(tracer, (:bysize, 300), A, y)
```
or similar. That way I can try a variety of selection mechanisms...